subhub/tests/unit/test_payments_mock.py

```diff
@@ -8,14 +8,14 @@
 import stripe
 
 from mockito import when, mock, unstub, ANY
-
+import pytest
```

Comment:
group `import blah` statements with others like `import stripe` above and group `from blah import blah` statements together in separate blocks; where you put `import pytest` doesn't follow this

Reply:
Good catch, updated.